### PR TITLE
feat: Export metadata

### LINKED ISSUE #27: [feature] Export metadat with the model
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind feature

**What happened**:

Now we just export the model from the local cache. we should also support exporting ormbfile.yaml from the local cache.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

/priority p0

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Export metadat when we call `ormb export xxx`

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #27 

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/oci/cache.go [hunk 1]
```diff
@@ -133,10 +137,20 @@ func (cache *Cache) FetchReference(ref *Reference) (*CacheRefSummary, error) {
 			if err != nil {
 				return &r, err
 			}
+			configBytes, err := cache.fetchBlob(r.Config)
```
Comment (simon-cj) [inline pkg/oci/cache.go:140]: I'm not understand how `Config` filed load and save model, the `Config` field looks like `Metadata` filed, why we need both of them simultaneously.

File: pkg/model/types.go [hunk 2]
```diff
@@ -6,19 +6,19 @@ type Model struct {
 	// Metadata is the contents of the Chartfile.
 	Metadata *Metadata `json:"metadata"`
 	Path     string    `json:"path"`
-	// TODO(gaocegege): Tar it in the code.
-	Content []byte `json:"content"`
+	Content  []byte    `json:"content"`
+	Config   []byte    `json:"content"`
```
[the remark this answers is not in the document]
Comment (gaocegege) [reply pkg/model/types.go:10]: Oh it is a type here.

File: pkg/oci/cache.go (hunk 1 above)
Comment (gaocegege) [reply pkg/oci/cache.go:140]: Maybe we will use it in the future, thus just keep the raw data here.

Comment (simon-cj) [reply pkg/oci/cache.go:140]: ok, LGTM

**Comment by gaocegege** [comment]
@simon-cj Comment is addressed PTAL

**Comment by ddysher** [comment]
@simon-cj you can add yourself to reviewers list (in the OWNERS file).

**Comment by simon-cj** [comment]
ok, I will add myself next time.

**Comment by gaocegege** [comment]
Did a rebase for the PR. PTAL

**Pull request merged**